src/app/route/MyAccountWishlist/MyAccountWishlist.style.scss

```diff
@@ -0,0 +1,77 @@
+.MyAccountWishlist {
+    --placeholder-animation: placeholderAnimation 2s linear infinite, appear 200ms forwards;
+    
+    &-Wrapper {
+        max-width: 1200px;
+        display: grid;
+        grid-template-columns: 20% 78%;
+        grid-column-gap: 2%;
+
+        padding: {
+            left: 1rem;
+            right: 1rem;
+        };
+
+        margin: {
+            top: 2rem;
+            left: auto;
+            right: auto;
+        }
+
+        @include mobile {
+            display: block;
```

Comment:
Here and in other cases, think about default element behavior. Maybe instead of `@include mobile { ... }` It is worth to use `@after-mobile { ... }`. This will allow not to repeat yourself in property declarations, like here, for example:

```scss
        display: grid;
        grid-template-columns: 20% 78%;
        grid-column-gap: 2%;

        @include mobile {
            display: block;
        }
```

Could be made without repeating `display` property:

```scss
        @include after-mobile {
                display: grid;
                grid-template-columns: 20% 78%;
                grid-column-gap: 2%;
        }
```

See other places in code, where the same logic can be applied.